### PR TITLE
Leaderboard - Project Setup

### PR DESCRIPTION
**Pull request by MaiRizk**
I have created an npm project with Webpack. 
The project is written with HTML and CSS only. 
This repo is built using Gitflow

### CODE REVIEW

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @MaiRizk , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project without a README.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: package.json [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    "style-loader": "^3.2.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.20.1",
+    "toml": "^3.0.0",
+    "webpack": "^5.52.0",
+    "webpack-cli": "^4.8.0",
+    "xml-loader": "^1.2.1",
+    "yamljs": "^0.3.0"
```
Comment (Georjane) [inline package.json]: - [ ] All webpack related packages should be included in package.json like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as devDependencies. You are missing `webpack-dev-server`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    "webpack-merge": "^5.8.0",
+    "webpack-sources": "^3.2.0",
+    "which": "^2.0.2",
+    "wildcard": "^2.0.0",
+    "yocto-queue": "^0.1.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```
Comment (Georjane) [inline package.json]: - [ ] npm run build throws an error. See the error below. Please fix this.
```
$ npm run build

> leaderboard@1.0.0 build
> webpack

[webpack-cli] Failed to load 'C:\Users\PC\Desktop\TSE\Leaderboard\webpack.config.js' config
[webpack-cli] Error: Cannot find module 'html-webpack-plugin'

```
- [ ] Also your npm should run scripts like the following. It does not need to be exact as long as there is a way to start the project say `npm run start`
```
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

**Review by Georjane — CHANGES_REQUESTED** [review]
### Hello, Team @MaiRizk, ✋🏽 

Good job so far! You have a very professional Readme file, your PR is professional and you have a good mastery of git-flow. Your linters are all passing, I applaud that too.
However, we have a few more changes to make in order to meet all requirements for this project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding! 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 🤝 

_Code Reviewer
**[Jane](https://witahgeorjane.tech/)**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by carlos-ssh — APPROVED** [review]
Hi @MaiRizk , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**